documentation/modules/rn-2.1.adoc

```diff
@@ -2,6 +2,7 @@
 //
 // * documentation/doc-Release_notes/master.adoc
 
+:_content-type: REFERENCE
```

Comment:
@sbeskin-redhat I'd call this a concept.

Reply:
Why is it a concept? It just lists the changes made in the new version.